oidc-samples/org.wso2.identity.sdk.oidc/org.wso2.identity.sdk.oidc/obj/org.wso2.identity.sdk.oidc.csproj.nuget.cache

```diff
@@ -0,0 +1,5 @@
+{
+  "version": 1,
+  "dgSpecHash": "Q4JtxSZaPcgDeEOnBoIo+COG/pbcUuBOCxR2v1VtDPkUSIJMQH0FVVFAEn44dSq0hDpgmo071lgsT8S6M5zuZw==",
+  "success": true
+}
```

Comment:
Add a new line at the end.

Reply:
Removed(This file generated while the build process).